fc/language/expressions/array.py

```diff
@@ -71,20 +40,20 @@ def Interpret(self, env):
     def DevelopResultWithCompile(self, range_name, ranges, compiled_gen_expr, env):
         local_dict = {}
         defined_ranges = 0
-        for i,name in enumerate(range_name):
+        for i, name in enumerate(range_name):
             local_dict[name] = np.array(ranges[i], dtype=float)
             defined_ranges += 1
         if defined_ranges > 1:
             raise NotImplementedError
-        
+
         result = ne.evaluate(compiled_gen_expr, local_dict, env.unwrappedBindings)
         return result
-        
+
     def DevelopResultWithInterpret(self, range_dims, range_name, ranges, env, num_gaps, dims):
-        #print 'Comprehension', self.location
+        # print 'Comprehension', self.location
```

Comment:
This is another flake8 (or actually pep?) rule that I'm happy to disable

Reply:
Most editors will comment with `# ` for you if you use the 'comment this block' shortcut...

Reply:
I'd rather just type a #  :D